amanuensis/blueprints/project.py

```diff
@@ -17,21 +16,49 @@
 
 logger = get_logger(__name__)
 
+
 # cache = SimpleCache()
 
 
+def determine_status_code(codes, consortium="Default"):
+    """
+    Takes status codes and returns the status name based on their precedence.
+    Example: if all request status are "APPROVED", then the status code will be "APPROVED".
+    However, if one of the request status is "PENDING", and "PENDING" has higher precedence
+    then the status code will be "PENDING".
+    """
+    try:
+        status_codes = config["CONSORTIUM_STATUS"][consortium]["Code"]
+        status_names = config["CONSORTIUM_STATUS"][consortium]["Name"]
```

Comment:
This need to up updated in the [config](https://github.com/chicagopcdc/amanuensis/blob/feature-PEDS-736-notify-CM-change-of-status/amanuensis/config-default.yaml).